gitlab/datadog_checks/gitlab/metrics.py

```diff
@@ -3,152 +3,211 @@
+    'geo_repositories_retrying_verification_count': 'geo.repositories_retrying_verification_count',
+    'geo_repositories_synced': 'geo.repositories_synced',
+    'geo_repositories_verification_failed': 'geo.repositories_verification_failed_count',
+    'geo_repositories_verification_failed_count': 'geo.repositories_verification_failed_count',
+    'geo_repositories_verified': 'geo.repositories_verified_count',
+    'geo_repositories_verified_count': 'geo.repositories_verified_count',
+    'geo_status_failed_total': 'geo_status_failed_total',
+    'geo_terraform_states': 'geo.terraform_states',
+    'geo_terraform_states_checksum_failed': 'geo.terraform_states_checksum_failed',
+    'geo_terraform_states_checksummed': 'geo.terraform_states_checksummed',
+    'geo_terraform_states_failed': 'geo.terraform_states_failed',
+    'geo_terraform_states_registry': 'geo.terraform_states_registry',
+    'geo_terraform_states_synced': 'geo.terraform_states_synced',
+    'geo_wikis_checksum_failed': 'geo.wikis_checksum_failed_count',
+    'geo_wikis_checksum_failed_count': 'geo.wikis_checksum_failed_count',
+    'geo_wikis_checksum_mismatch': 'geo.wikis_checksum_mismatch_count',
+    'geo_wikis_checksum_mismatch_count': 'geo.wikis_checksum_mismatch_count',
+    'geo_wikis_checksummed': 'geo.wikis_checksummed_count',
+    'geo_wikis_checksummed_count': 'geo.wikis_checksummed_count',
+    'geo_wikis_retrying_verification': 'geo.wikis_retrying_verification_count',
+    'geo_wikis_retrying_verification_count': 'geo.wikis_retrying_verification_count',
+    'geo_wikis_verification_failed': 'geo.wikis_verification_failed_count',
+    'geo_wikis_verification_failed_count': 'geo.wikis_verification_failed_count',
+    'geo_wikis_verified': 'geo.wikis_verified_count',
+    'geo_wikis_verified_count': 'geo.wikis_verified_count',
+    'gitlab_banzai_cached_render_real_duration_seconds': 'banzai.cached_render_real_duration_seconds',
+    'gitlab_banzai_cacheless_render_real_duration_seconds': 'banzai.cacheless_render_real_duration_seconds',
+    'gitlab_cache_misses_total': 'cache.misses_total',
+    'gitlab_cache_operation_duration_seconds': 'cache.operation_duration_seconds',
+    'gitlab_cache_operations_total': 'cache_operations_total',
+    'gitlab_ci_pipeline_creation_duration_seconds': 'ci_pipeline_creation_duration_seconds',
+    'gitlab_ci_pipeline_size_builds': 'ci_pipeline_size_builds',
+    'gitlab_database_connection_pool_busy': 'database_connection_pool_busy',
+    'gitlab_database_connection_pool_connections': 'database_connection_pool_connections',
+    'gitlab_database_connection_pool_dead': 'database_connection_pool_dead',
+    'gitlab_database_connection_pool_idle': 'database_connection_pool_idle',
+    'gitlab_database_connection_pool_size': 'database_connection_pool_size',
+    'gitlab_database_connection_pool_waiting': 'database_connection_pool_waiting',
+    'gitlab_database_transaction_seconds': 'database.transaction_seconds',
```

Comment:
Here there's one metric that uses `database.` while the rest uses `database_`. I suppose we should stick to `database.`?

```suggestion
    'gitlab_database_connection_pool_busy': 'database.connection_pool_busy',
    'gitlab_database_connection_pool_connections': 'database.connection_pool_connections',
    'gitlab_database_connection_pool_dead': 'database.connection_pool_dead',
    'gitlab_database_connection_pool_idle': 'database.connection_pool_idle',
    'gitlab_database_connection_pool_size': 'database.connection_pool_size',
    'gitlab_database_connection_pool_waiting': 'database.connection_pool_waiting',
    'gitlab_database_transaction_seconds': 'database.transaction_seconds',
```

Reply:
Yes, that example is the opposite of the first one. The pre-existing metric (`database.transaction_seconds`) has a `database` namespace that we should reuse. Thanks for catching